pkmodel/model.py

```diff
@@ -4,14 +4,105 @@
 
 class Model:
     """A Pharmokinetic (PK) model
+    This model sets up the right hand side of the system of ODEs
```

Comment:
This is not the goal of this class at the moment. This model only stores data now.